src/containers/Application.jsx

```diff
@@ -55,6 +55,12 @@ import WhiteListProvider, {
 
 import '../lib/validators';
 
+import CampaginCreateNew from '../components/views/CampaginCreateNew';
```

Comment:
It should be renamed to MilestoneCreateOptionsMenu, or something like that which you prefer